flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/source/ArcticScanContext.java

```diff
@@ -49,6 +51,8 @@ public class ArcticScanContext extends ScanContext implements Serializable {
   protected ArcticScanContext(
       boolean caseSensitive,
       Long snapshotId,
+      StreamingStartingStrategy startingStrategy,
```

Comment:
does it duplicate with `scanStartupMode`?

Reply:
There is no duplication between them. The scanStartupMode parameter is an Arctic-defined parameter that controls the startup method with the primary key table read; the startingStrategy is an Iceberg-native parameter that is only passed to the Iceberg ScanContext constructor in the ArcticScanContext class and is not actually used.